tsconfig.json

```diff
@@ -31,7 +31,6 @@
     "src/internal",
     "src/**/*.test.tsx",
     "src/**/*.stories.tsx",
-    "tst",
```

Comment:
[Understanding: ]We are not running tslint or eslint on tests right? Are we removing it from `typescript` checking too? Should we keep it in the `tsconfig` as it would benefit having typescript checking for tests. I checked but I do not see any test files under `src` hence checking once.